SU-HSDO/suhumsci
# Hsd8 386 manage content

# READY FOR REVIEW

# Summary
- Added a manage content view similar to what was on D7

# Needed By (Date)
- 11/7

# Urgency
- low

# Steps to Test
1. Checkout this branch
1. sync a site
1. verify the admin menu under "Content" now contains "Manage Content"
1. click on that menu item
1. verify there are tabs for each of the content types
1. verify each of those tabs displays appropriate content.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
